packages/components/src/LoaderApp/LoaderApp.component.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function LoaderApp() {
```

Comment:
Why isn't this included inside the Loader folder as a sub-component?
It you make more sense as type of loader.
I would also add that I'm not OK with the name `LoaderApp`. It gives the impression that it's an "application loader" (as in "plugin") rather than an application loading screen.

Reply:
Because it serve a complete different purpose
I'm open to any name to describe it better.
In fact I don't understand why do we have Loader at the moment which is just CircularPropgress nothing more.
In our case we have a structure with a CSS taht has nothing to do with the current Loader component.